README.md

```diff
@@ -17,28 +17,32 @@
 
-The research applications group is led by [Rey Granillo](https://github.com/reyg3) Director of Technology and Research Computing, [Leland Boeman](https://github.com/lboeman) Research & Development Software Engineer, and [Thomas Weiss](https://github.com/tweissaz) Research & Development Systems Engineer.
+TRAD is led by [Rey Granillo](https://github.com/reyg3) Director of Technology and R&D Engineering, [Leland Boeman](https://github.com/lboeman) Research & Development Software Engineer, and [Thomas Weiss](https://github.com/tweissaz) Research & Development Systems Engineer.
 
 ### Our proficiencies
-- Research focused software development
-- Cloud computing
+- Research & Development / Proof of Concept
+  - Research-focused software development
+  - Hardware and sensor development
+- Cloud computing infrastructure
+  - Amazon Web Services (AWS)
+  - Google Cloud Platform (GCP)
 - Data assimilation, processing, and presentation
 - Database architecture and development
 - Application Programming Interface (API) development
 - Remote sensing 
-  - LoRaWAN network gateways
-  - Designing and building microcontrollers
+  - LoRaWAN
+  - Microcontroller design and development
 - 3D printing
-- Machine learning
+- Machine Learning (ML)
 
 ### How we do it
 #### Development Process
-By developing and combining technologies, we incorporate novel technical solutions into new and existing research. Using a collaborative model, versus the typical service model found most in Information Technology, we can better understand project data, the research questions being asked, and provide higher quality research collaborations. Having that greater understanding has led us to find correlations in data that were not previously realized which have led to new research project ideas and proposals. We collaborate with researchers during the proposal process and can provide a proof-of-concept application or tool showing what is technically feasible if project funding is awarded. Upon project completion, we aid with the technical writing portions of publications which outline our findings and technical solutions that were implemented.
+By developing and combining technologies, we incorporate novel technical solutions into new and existing research. Using a collaborative model, we can better understand project data, the research questions being asked, and provide higher quality research collaborations. Having that greater understanding has led us to find correlations in data that were not previously realized which have led to new research project ideas and proposals. We collaborate with researchers during the proposal process and can provide a proof-of-concept application or tool showing what is technically feasible if project funding is awarded. Upon project completion, we aid with the technical writing of publications which outline our findings and technical solutions that were implemented.
```

Comment:
```suggestion
Using a collaborative model, TRAD members gain a broader understanding of the research questions being asked, and collaborators gain a better understanding of how cutting edge technology can be applied to their area of expertise. These collaborations have uncovered correlations in data that were not previously realized which have led to new research project ideas and proposals. We collaborate at each step of the research process from proposal development to publication. TRAD can provide a proof-of-concept application or tool showing what is technically feasible to add a competitive edge to proposals. Upon project completion, we aid with the technical writing of publications which outline our findings and technical solutions that were implemented.
```
What do you think about this rework? I felt like some of the points read a little awkwardly and tried to make it more direct.